Mixpanel/MPNotificationViewController.m

```diff
@@ -147,7 +147,7 @@ - (UIStatusBarAnimation)preferredStatusBarUpdateAnimation
     return UIStatusBarAnimationFade;
 }
 
-- (UIInterfaceOrientationMask)supportedInterfaceOrientations
+- (NSUInteger)supportedInterfaceOrientations
```

Comment:
- WARN  | [Mixpanel/Mixpanel]  /Users/samgreen/code/mixpanel-iphone/Mixpanel/MPNotificationViewController.m:150:1: warning: conflicting return type in implementation of 'supportedInterfaceOrientations': 'NSUInteger' (aka 'unsigned long') vs 'UIInterfaceOrientationMask' (aka 'enum UIInterfaceOrientationMask') [-Wmismatched-return-types]